tests/main/snap-download/task.yaml

```diff
@@ -33,3 +34,8 @@ execute: |
     su -l -c "SNAPPY_USE_STAGING_STORE=$SNAPPY_USE_STAGING_STORE HTTPS_PROXY=$HTTPS_PROXY snap download test-snapd-tools" test
     ls /home/test/test-snapd-tools_*.snap
     verify_asserts /home/test/test-snapd-tools_*.assert
+
+    echo "Snap download can download snaps from a cohort"
+    snap download --cohort="MSBlRmU4QlRSNUw1VjlGN3lIZU1BUHhrRXIyTmRVWE10dyAxNTU1NTE0MzA5IDg3ZmUwMjhkZDFjMTQ1MDY5N2QyZjdiMGZkMzgzODk0NjMzMmFhOTZmZmFjZmFlNmU2MGQyOTNjYzE1OTE3NWY=" test-snapd-tools
```

Comment:
Can we also test for errors if snap download is given revision and cohort at the same time? What about if a cohort key is invalid, can we have a test for respective error here?

Reply:
You can't specify revision without being logged in, so I can't really check this from spread.
I'll add one for invalid cohort key.

Reply:
actually i can't add one for that yet because the fix for it hasn't yet been deployed

Reply:
fix got deployed, test added

Reply:
Thank you!